dot/network/sync.go

```diff
@@ -356,6 +358,28 @@ func (q *syncQueue) benchmark() {
 	}
 }
 
+func (q *syncQueue) sentBlockIntervalTelemetry() {
```

Comment:
this function could be just part of the `Service`, since it doesn't use any `syncQueue` functionality

Reply:
Good idea, I've moved this.